components/sanity/scripts/start.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
```

Comment:
Just as a general remark here.. please make sure this is all idempotent. If the sanity fails but the upgrade was successful and we perform a rollback only because of a single failure here which could be repeated by an operation retry..it would .. suck.. to say the least